### PR TITLE
Update Release-Notes.md

### PR DESCRIPTION
**Pull request by rich-howell**
Release note prep for next release.

### CODE REVIEW

File: docs/Release-Notes.md [hunk 1]
```diff
@@ -2,6 +2,89 @@
...
+* [#570](https://github.com/actualbudget/actual/pull/570) Use navigitor.userAgent to determine isMobile -- thanks [shall0pass](https://github.com/shall0pass)
+* [#573](https://github.com/actualbudget/actual/pull/573) Goal templates -- thanks [shall0pass](https://github.com/shall0pass)
+* [#579](https://github.com/actualbudget/actual/pull/579) Add 'View on Hover' to Category Notes for #563 -- thanks [gsumpster](https://github.com/gsumpster)
+* [#580](https://github.com/actualbudget/actual/pull/580) Added date to export file name -- thanks [rich-howell](https://github.com/MatissJanis)
+* [#584](https://github.com/actualbudget/actual/pull/585) Cover Overspending dropdown menu, open on click -- thanks [shall0pass](https://github.com/shall0pass)
+* [#590](https://github.com/actualbudget/actual/pull/590) Add support for filtering the schedules table -- thanks [j-f1](https://github.com/j-f1)
+* [#591](https://github.com/actualbudget/actual/pull/591) Allow libofx to handle decoding imported files -- thanks [j-f1](https://github.com/j-f1)
```
Comment (MatissJanis) [inline docs/Release-Notes.md:30]: This one's a bugfix.

File: docs/Release-Notes.md [hunk 2]
```diff
@@ -2,6 +2,89 @@
...
+* [#592](https://github.com/actualbudget/actual/pull/592) Update SelectedBalance to use useSheetValue -- thanks [j-f1](https://github.com/j-f1)
```
Comment (MatissJanis) [inline docs/Release-Notes.md:31]: Another bugfix

File: docs/Release-Notes.md [hunk 3]
```diff
@@ -2,6 +2,89 @@
...
+* [#593](https://github.com/actualbudget/actual/pull/593) Allow creating a payee with a name matching an existing account -- thanks [j-f1](https://github.com/j-f1)
+* [#598](https://github.com/actualbudget/actual/pull/595) Allow configuring the server from any page on the management app -- thanks [j-f1](https://github.com/j-f1)
+* [#600](https://github.com/actualbudget/actual/pull/600) Add a warning when SharedArrayBuffer is not available -- thanks [j-f1](https://github.com/j-f1)
+* [#601](https://github.com/actualbudget/actual/pull/601) Improve handling of schedules that are missing a date -- thanks [j-f1](https://github.com/j-f1)
```
Comment (MatissJanis) [inline docs/Release-Notes.md:35]: Bugfix

File: docs/Release-Notes.md [hunk 4]
```diff
@@ -2,6 +2,89 @@
 title: Release Notes
 ---
 
+## 23.2.5
+**Docker tag: 23.2.5**
+
+The release has notable of improvements of:
```
Comment (MatissJanis) [inline docs/Release-Notes.md:8]:
```suggestion
The release has notable security improvements. Highlights:
```

File: docs/Release-Notes.md [hunk 5]
```diff
@@ -2,6 +2,91 @@
...
+
+* [#109](https://github.com/actualbudget/actual/pull/109) fix: listen also listen on ipv6 addresses -- thanks [heilerich](https://github.com/heilerich)
+
+#### Maintenance
+
+* [#116](https://github.com/actualbudget/actual/pull/116) 🔥 remove unused code (plaid, sync-full) -- thanks [MatissJanis](https://github.com/MatissJanis)
+* [#110](https://github.com/actualbudget/actual/pull/110) build(deps): bump node-fetch from 2.2.0 to 2.6.7
+* [#111](https://github.com/actualbudget/actual/pull/111) build(deps): bump minimatch from 3.0.4 to 3.1.2
+* [#112](https://github.com/actualbudget/actual/pull/112) build(deps): bump moment from 2.29.3 to 2.29.4
+v[#117](https://github.com/actualbudget/actual/pull/117) build(deps): bump http-cache-semantics from 4.1.0 to 4.1.1
```
Comment (j-f1) [inline docs/Release-Notes.md:86]:
```suggestion
* [#117](https://github.com/actualbudget/actual/pull/117) build(deps): bump http-cache-semantics from 4.1.0 to 4.1.1 
```

File: docs/Release-Notes.md [hunk 6]
```diff
@@ -2,6 +2,91 @@
...
+* [#584](https://github.com/actualbudget/actual/pull/585) Cover Overspending dropdown menu, open on click -- thanks [shall0pass](https://github.com/shall0pass)
+* [#590](https://github.com/actualbudget/actual/pull/590) Add support for filtering the schedules table -- thanks [j-f1](https://github.com/j-f1)
+* [#593](https://github.com/actualbudget/actual/pull/593) Allow creating a payee with a name matching an existing account -- thanks [j-f1](https://github.com/j-f1)
+* [#598](https://github.com/actualbudget/actual/pull/595) Allow configuring the server from any page on the management app -- thanks [j-f1](https://github.com/j-f1)
+* [#600](https://github.com/actualbudget/actual/pull/600) Add a warning when SharedArrayBuffer is not available -- thanks [j-f1](https://github.com/j-f1)
+* [#601](https://github.com/actualbudget/actual/pull/601) Improve handling of schedules that are missing a date -- thanks [j-f1](https://github.com/j-f1)
+* [#602](https://github.com/actualbudget/actual/pull/602) Support arbitrary currency symbols in expressions -- thanks [j-f1]
```
Comment (j-f1) [inline docs/Release-Notes.md:34]:
```suggestion
* [#602](https://github.com/actualbudget/actual/pull/602) Support arbitrary currency symbols in expressions -- thanks [j-f1](https://github.com/j-f1)
```

File: docs/Release-Notes.md [hunk 7]
```diff
@@ -2,6 +2,91 @@
...
 ---
 
+## 23.2.5
+**Docker tag: 23.2.5**
+
+The release has notable security improvements. Highlights:
+
+* e2e encryption
+* login invalidation when changing password
+* dependency upgrades to solve actual-server security issues
```
Comment (j-f1) [inline docs/Release-Notes.md:12]:
```suggestion
* dependency upgrades to remove potential actual-server security vulnerabilities (although we don’t believe there were any severe issues)
```

I want this to balance the line of confirming we’ve made improvements without making people worried that there were serious vulnerabilities before (unless you’re aware of any)

**Review by MatissJanis — APPROVED** [review]

**Review by j-f1 — APPROVED** [review]
🚀

**Pull request merged**